### PR TITLE
Adding student/nate

### PR DESCRIPTION
**Pull request by nathanieltornow**

### CODE REVIEW

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors cancel

File: modules/users/students.nix [hunk 1]
```diff
@@ -336,6 +340,15 @@ in
       uid = 2026;
       openssh.authorizedKeys.keys = kamilKeys;
     };
+    
+    nate = {
```
Comment (Mic92) [inline modules/users/students.nix:344]: Are you currently employed by university on a position or is this for a thesis?

Comment (nathanieltornow) [reply modules/users/students.nix:344]: Yes, I'm doing the Quantum project and am a tutor

Comment (Mic92) [inline modules/users/students.nix:344]:
```suggestion
    #  Nathaniel Tornow, working on the Quantum project + tutor
    nate = {
```

**Comment by Mic92** [comment]
bors merge

File: modules/users/students.nix [hunk 2]
```diff
@@ -341,6 +344,16 @@ in
...
       openssh.authorizedKeys.keys = kamilKeys;
     };
+    
+    #  Nathaniel Tornow, working on the Quantum project + tutor
+    nate = {
+      isNormalUser = true;
+      home = "/home/nate";
+      inherit (config.users.users.joerg) extraGroups;
+      shell = "/run/current-system/sw/bin/zsh";
+      uid = 2027;
```
Comment (Mic92) [inline modules/users/students.nix:354]:
```suggestion
      uid = 2028;
```

**Comment by Mic92** [comment]
bors merge

**Pull request merged**